roxie/ccd/ccdserver.cpp

```diff
@@ -13471,6 +13670,21 @@ class CRoxieServerThreadedConcatActivity : public CRoxieServerActivity
         CRoxieServerActivity::stop();
     }
 
+    virtual RoxieSourceCharacteristics getAllInputsCharacteristics() const override
+    {
+        RoxieSourceCharacteristics ret = RSC::none;
+        ForEachItemIn(idx, pullers)
+            ret |= pullers.item(idx).queryInput()->getSourceCharacteristics();
+        return ret;
+    }
+
+
+    virtual RoxieSourceCharacteristics getActivityCharacteristics() const override
+    {
+        // MORE - should this be urgent? Does it really have latency ?
```

Comment:
Not sure about this one.  I don't think it should default to having row latency.  Possibly urgent, but not sure.